tests/integration/store/query-test.js

```diff
@@ -39,7 +39,7 @@ test("meta is proxied correctly on the PromiseArray", function(assert) {
     result = store.query('person', {});
   });
 
-  assert.equal(result.get('meta.foo'), undefined);
+  assert.strictEqual(result.get('meta.foo'), undefined);
```

Comment:
@pangratz I spent some time digging into this and couldn't figure out while its failing. I blame phantom weirdness. Lets leave this as `assert.equal` so the tests pass since the fact that this returns undefined is Ember behavior and not Ember Data.

Reply:
Correction, I was just able to reproduce it. Turns out in Ember 1.13 `Ember.get({foo: null}, 'foo.bar')` returns `null` but in Ember 2.0 `Ember.get({foo: null}, 'foo.bar')` returns `undefined`.

Reply:
Uff, thanks for tracking that one down!

I've updated this assertion to be `assert.notOk`.